src/docs/references/transaction-process-actions/index.md

```diff
@@ -799,6 +799,120 @@ Decline a pending or proposed booking.
 
 **Configuration options**: -
 
+### Stock reservations
+
+#### :action/create-pending-stock-reservation
+
+Creates a new stock reservation in `pending` state with given stock, as
+long as the listing's stock permits the reservation to be created.
+
+A stock reservation in `pending` state decreases the listing's stock - a
+stock adjustment is created having the given `stockReservationQuantity`
+(with a negative sign).
+
+There can be at most one stock reservation per transaction.
+
+**Preconditions**:
+
+- The transaction must not already have a stock reservation.
+- The listing must have sufficient stock available.
+
+**Parameters**:
+
+- `stockReservationQuantity`: positive integer, mandatory.
+
+  The quantity of stock that is reserved from a listing's stock.
+
+**Configuration options**: -
+
+#### :action/create-proposed-stock-reservation
+
+Creates a new stock reservation in `proposed` state with given stock, as
+long as the listing's stock permits the reservation to be created.
+
+A stock reservation in `proposed` state does not affect the listing's
+available stock, i.e. the listing's stock is not decreased by the
+proposed reservation quantity until the reservation is accepted using
+the `:action/accept-stock-reservation` action.
+
+There can be at most one stock reservation per transaction.
+
+**Preconditions**:
+
+- The transaction must not already have a stock reservation.
+- The listing must have sufficient stock available.
+
+**Parameters**:
```

Comment:
```suggestion
**Parameters**: -
```

Although looking at the rest of the file, this may not create a line break? (for example, look at https://github.com/sharetribe/flex-docs/blob/59e30409379d2b8471cb4ba8a121162e09fa3b6d/src/docs/references/transaction-process-actions/index.md#actionstripe-confirm-payment-intent, it shows `Parameters: - Configurations: -`).